src/DataTable/index.tsx

```diff
@@ -62,6 +62,7 @@ export interface DataTableProps<T extends Record<string, unknown>>
     tableToolbar?: (
       props: TableToolbarProps,
     ) => ReactElement,
+    editingRow?: boolean,
```

Comment:
Let's use isEditing here so that we can stick with a better naming convention for booleans.